engine/core/src/main/java/org/datacleaner/util/convert/ResourceConverter.java

```diff
@@ -94,9 +104,24 @@ public String getScheme() {
 
     /**
      * Constructs a {@link ResourceConverter} using a default set of handlers
+     * 
+     * @deprecated use another constructor instead
```

Comment:
I general I would expect how to get the "most equivalent" one in the simplest way, in this case ```new ResourceConverter(new DataCleanerConfigurationImpl())```, but yeah, it's should be easy to pick one :)